### PR TITLE
MSGF+ min_peaks option added

### PR DESCRIPTION
**Pull request by ypriverol**
## Description

MSGF+ supports a minimum number of peaks for spectra to be analyzed. Here, in this PR we include the option to make it more consistent and standardised with SAGE and COMET.   

## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out

- The details of the action statuses at the end of the PR or the "Checks" tab.
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. Use the "Show filters" capability on the top right to search for your PR number.
  If you click in the column that lists the failed tests you will get detailed error messages.

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

**Comment by ypriverol** [comment]
@timosachsenberg what is the error here?

File: src/topp/MSGFPlusAdapter.cpp [hunk 1]
```diff
@@ -174,6 +174,9 @@ class MSGFPlusAdapter :
     registerIntOption_("matches_per_spec", "<num>", 1, "Number of matches per spectrum to be reported (MS-GF+ parameter '-n')", false);
     setMinInt_("matches_per_spec", 1);
 
+    registerIntOption_("min_peaks", "<num>", 10, "Minimum number of ions a spectrum must have to be examined");
```
Comment (timosachsenberg) [inline src/topp/MSGFPlusAdapter.cpp]:
```suggestion
    registerIntOption_("min_peaks", "<num>", 10, "Minimum number of ions a spectrum must have to be examined", false); 
```

Comment (timosachsenberg) [reply src/topp/MSGFPlusAdapter.cpp]: makes it an optional option with 10 as default

**Comment by ypriverol** [comment]
@timosachsenberg in the previous actions was difficult for me to see where everything was failing, can you let me know? I want to start contributing slowly to small details like this in OpenMS but I don't want to work more than me ;)

**Comment by timosachsenberg** [comment]
https://cdash.openms.de/index.php?project=OpenMS
there is a link hidden in the action output but I usually go through this page

**Pull request merged**